kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/internal/active.kt

```diff
@@ -1,98 +1,107 @@
 package io.kotest.core.internal
 
+import io.kotest.core.config.ExperimentalKotest
 import io.kotest.core.config.configuration
-import io.kotest.core.extensions.IsActiveExtension
+import io.kotest.core.extensions.EnabledExtension
+import io.kotest.core.extensions.resolvedExtensions
 import io.kotest.core.filter.TestFilter
 import io.kotest.core.filter.TestFilterResult
-import io.kotest.core.test.TestCase
-import io.kotest.core.test.isBang
-import io.kotest.core.test.isFocused
-import io.kotest.core.spec.focusTests
 import io.kotest.core.internal.tags.activeTags
 import io.kotest.core.internal.tags.allTags
 import io.kotest.core.internal.tags.isActive
 import io.kotest.core.internal.tags.parse
 import io.kotest.core.plan.toDescriptor
+import io.kotest.core.spec.focusTests
+import io.kotest.core.test.Enabled
+import io.kotest.core.test.TestCase
 import io.kotest.core.test.TestCaseSeverityLevel
+import io.kotest.core.test.isBang
+import io.kotest.core.test.isFocused
 import io.kotest.mpp.log
 import io.kotest.mpp.sysprop
 
 /**
- * Returns true if the given [TestCase] is active based on default rules at [isActiveInternal]
- * or any registered [IsActiveExtension]s.
+ * Returns [Enabled.isEnabled] if the given [TestCase] is active based on default rules at [isEnabledInternal]
+ * or any registered [EnabledExtension]s.
  */
-suspend fun TestCase.isActive(): Boolean {
+@OptIn(ExperimentalKotest::class)
+suspend fun TestCase.isEnabled(): Enabled {
```

Comment:
This is a public function I know people could be using.
Can you add a forwarding function with the old name.

Reply:
Lets make the new one internal.

Reply:
ah can't make it internal cause other modules need it